src/main/java/racingcar/domain/History.java

```diff
@@ -0,0 +1,28 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class History {
+
+    private List<List<Car>> carsHistory;
```

Comment:
일급 컬렉션인 `Cars`를 사용해 보세요.

```suggestion
    private List<Cars> carsHistory;
```

Reply:
네, Cars로 대체할 수 있었다는 사실을 미처 알지 못했었습니다.
피드백 감사합니다!